packages/atomic/src/utils/jsx-utils.tsx

```diff
@@ -0,0 +1,44 @@
+import {Fragment, FunctionalComponent, h, VNode} from '@stencil/core';
+import {AnyBindings} from '../components/common/interface/bindings';
+
+export interface LocalizedStringProps {
+  bindings: AnyBindings;
+  key: string;
+  params: Record<string, VNode | string>;
+}
+
+export const LocalizedString: FunctionalComponent<LocalizedStringProps> = ({
+  bindings,
+  key,
+  params,
+}) => {
+  const delimitingCharacter = '\u001d'; // Unicode group separator
+  const placeholderPrefixCharacter = '\u001a'; // Unicode substitute character
```

Comment:
I used unicode control characters since I don't expect those to be used in any locales.